app/locale/tr_TR/IyzicoCheckoutForm_IyzicoCheckoutForm.csv

```diff
@@ -0,0 +1,27 @@
+Enabled,Enabled
+Activate Testmode,Mode: Test/Live
+TestApiId,TEST API ID
+TestSecretKey,TEST SECRET KEY
+LiveApiId,LIVE API ID
+LiveSecretKey,LIVE SECRET KEY
+Iyzico Basic Setting,iyzico ayarları 
+Error before redirect,Bir hata oluştu. İşlem sırasında bir hata oluştu.
+This page is not allowed at this time.,Bir hata oluştu. Sayfayı acma yetkiniz yok.
+Please make sure that you enter a secure SSL URL.,Bir hata oluştu. SSL'nin etkin olduğundan emin olun.
+SortOrderTxt,Sort Order,
+kredi kartı (powered by iyzico),kredi kartı (powered by iyzico)
+Live, Live,
+Test, Test,
+orderID,siparis no,
+Credit Card,Kredi Kartı
+transaction marked as invalid,işlem geçersiz olarak işaretlenir
+This email is already registered as a Iyzico merchant.," ""Bu e-posta zaten Iyzico tüccar olarak tescil edilmiştir."""
+Registration successful. Keys fetched and filled successfully. Please hit Save button to save the settings.," ""Kayıt başarılı. Tuşlar getirilen ve başarıyla dolu. Ayarları kaydetmek için Kaydet tuşuna basın lütfen."""
```

Comment:
Kayıt başarılı. Anahtarlar alındı ve başarıyla dolduruldu. Lütfen kaydetmek için Kaydet butonuna basın.